omod/src/main/java/org/openmrs/module/owa/activator/OwaActivator.java

```diff
@@ -44,7 +44,7 @@ public void contextRefreshed() {
 		 */
 		String owaAppFolderPath = Context.getAdministrationService().getGlobalProperty(AppManager.KEY_APP_FOLDER_PATH);
 		if (null == owaAppFolderPath) {
-			owaAppFolderPath = OpenmrsUtil.getApplicationDataDirectory() + "owa";
+			owaAppFolderPath = OpenmrsUtil.getApplicationDataDirectory() + "/owa";
```

Comment:
@psbrandt : Might be better to use - `OpenmrsUtil.getApplicationDataDirectory() + File.separator + "owa"` for cross-platform purposes?